src/main/grammars/RustParser.bnf

```diff
@@ -1524,6 +1524,20 @@ LetDecl ::= OuterAttr* let Pat TypeAscription? [ '=' AnyExpr ] ';' {
 
 EmptyStmt ::= ';' { extends = Stmt }
 
+///////////////////////////////////////////////////////////////////////////////////////////////////
+// REPL
+///////////////////////////////////////////////////////////////////////////////////////////////////
+
+// copy of BlockElement with '}' replaced to <<eof>>
```

Comment:
I'm not really sure why `<<eof>>` is needed here, but maybe it is. @dima74 @vlad20012 what do you think?

Reply:
Looks like `<<eof>>` is not needed in `ReplBlockElement`. But it is needed in `ReplExprStmtOrLastExpr` (otherwise tail expression will not be parsed correctly)

Reply:
It's ok for `ReplExprStmtOrLastExpr`